### ISSUE TITLE
Modify all check if that need specific Odoo version

### ISSUE THREAD

**Issue by JesusZapata** [OCA/pylint-odoo#180]
Modify all check that depend of the Odoo version to use the new way to skip the check if that check fails with the Odoo version

For example the check `your-new-check` only run over the Odoo `9.0` and `10.0` within the class that defines the check `odoo_check_versions` to skip the check if the version is not equal to `9.0` or `10.0`

```python
...
class FormatChecker(PylintOdooTokenChecker):
...
    odoo_check_versions = {
        'your-new-check': {
            'min_odoo_version': '9.0',
            'max_odoo_version': '10.0',
        }
    }
...

```

**Comment by moylop260** [comment]
Closing since that PR related is closed too.
If a check requires a particular odoo version we should open a new issue for that case.
